### ISSUE TITLE
FilterExpression doesn't work with LocalDateTime values

### ISSUE THREAD

**Issue by pinheirosam** [intuit/graphql-filter-java#29]
When passing in a LocalDateTime, the expression parser needlessly converts into a `Date`

<img width="789" alt="image" src="https://user-images.githubusercontent.com/57364583/217609210-e726359d-f25a-44ad-8168-9fef7393094f.png">

<img width="717" alt="image" src="https://user-images.githubusercontent.com/57364583/217609111-ccfe6e0d-19f4-406f-8a3b-cd6c47797d11.png">

This makes the filter unusable if you want to convert it to JPA format or SQL using `filterExpression.getExpression(ExpressionFormat.JPA);`

**Comment by PrabhatUpadhyay1** [comment]
Is it still open?

**Comment by pinheirosam** [comment]
@PrabhatUpadhyay1 Yes. We had to publish our own version of this library to use this locally. But ideally this fix could be merged and released
